.travis.yml

```
@@ -24,10 +24,23 @@ install:
   # Install extra requirements for running tests and building docs.
   - pip install -r requirements-dev.txt
 
+
+env:
+  global:
+    # Doctr deploy key for NSLS-II/amostra
```

The key is for the NSLS-II/NSLS-II.github.io repository. It deploys into an amostra _directory_ (as specified by the end of the `doctr deploy ...` command below) but this line should specific the _repo_. See https://github.com/bluesky/bluesky/blob/23e81a30aacd68b2002fbf5836b70a05e3e50ae7/.travis.yml#L7 for comparison.

```suggestion
    # Doctr deploy key for NSLS-II/NSLS-II.github.io
```